src/mpdevil.py

```diff
@@ -759,7 +761,15 @@ def _to_playlist(self, append, mode):  # modes: play, append, enqueue
 					self.delete(int(duplicates[1]["pos"])-1)
 
 	def files_to_playlist(self, files, mode):
-		def append():
+		def append(next=False):
+			if next:
+				status=self.status()
+				try:
+					for f in files:
+						self.addid(f, int(status["song"])+1)
```

Comment:
This is where the implicit reverse happens.

Reply:
`for f in files.reverse():` should do it?

Reply:
Yes.